vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewModel.kt

```diff
@@ -74,11 +79,16 @@ class SessionOverviewViewModel @AssistedInject constructor(
     }
 
     init {
+        refreshPushers()
         observeSessionInfo(initialState.deviceId)
         observeCurrentSessionInfo()
         observePushers(initialState.deviceId)
     }
 
+    private fun refreshPushers() {
+        session.pushersService().refreshPushers()
```

Comment:
Could you remove the `session` parameter from the `ViewModel` and replace it by using the existing `activeSessionHolder`?. As a general rule, it is safer to inject and use this holder.